addons/web/static/src/views/fields/many2one/many2one_field.js

```diff
@@ -194,7 +194,7 @@ export class Many2OneField extends Component {
         }
     }
     get extraLines() {
-        return this.value
+        return this.value && this.value[1]
```

Comment:
you should backport the test as well (first commit)

Reply:
Backport? In which version ?

Reply:
the first commit is basically a backport from master, but you only backport the fix, not the test